5/alpine/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# https://docs.ghost.org/faq/node-versions/
+# https://github.com/nodejs/Release (looking for "LTS")
+# https://github.com/TryGhost/Ghost/blob/v4.1.2/package.json#L38
+FROM node:16-alpine3.14
```

Comment:
```suggestion
# https://github.com/TryGhost/Ghost/blob/v5.0.0/package.json#L54
FROM node:16-alpine3.15
```
Update link and get a slightly newer Alpine release. I'd go for Alpine 3.16, but there isn't a node image of that yet.